onnxruntime/core/providers/tensorrt/tensorrt_execution_provider.cc

```diff
@@ -353,8 +407,36 @@ common::Status TensorrtExecutionProvider::Compile(const std::vector<onnxruntime:
     }
 
     trt_builder->setMaxBatchSize(max_batch_size_);
-    trt_builder->setMaxWorkspaceSize(max_workspace_size_);
-    auto trt_engine = unique_pointer<nvinfer1::ICudaEngine>(trt_builder->buildCudaEngine(*trt_network.get()));
+    std::cout << "max_batch_size_: " << max_batch_size_ << std::endl;
```

Comment:
we shouldn't have cout statements.
did you mean to use log statements? or should it be removed entirely?